### ISSUE TITLE
need a sample example without vuejs

### ISSUE THREAD

**Issue by omarsedki** [newcat/baklavajs#63]
It look like a perfect editor node.  But the documentation still very hard to follow . Can you put a simple example code to  help us understand the basic for the library.
Thanks

**Comment by newcat** [comment]
Yes I can do that. I try to make it happen in the next few days, but I can't promise. May also take one or two weeks, I am currently very busy unfortunately.

**Comment by newcat** [comment]
I have updated the documentation to include a full HTML skeleton of a page using Baklava: https://newcat.github.io/baklavajs

It's good that you created this issue because I found some bugs which made the previous version of the bundle pretty much unusable. Thank you 😄 

I am going to close this issue; however, if you have any further questions, feel free to reopen and ask them.
